src/md/release.md

```diff
@@ -201,6 +252,27 @@ git push apache
 ```
 Merge `@since` tag and change log changes to `master`.
 
+### Documentation
+
+The documentation promotion is done in two steps. First the release documentation component is made available on the website and then the remaining website documentation is deployed.
+
+1. If this is a new minor or a major release, in the `apex-site` folder from the build, run the following commands to point the website to the release documentation folder, otherwise it is a patch release and this step can be skipped as the documentation is already reflected in the website.
+
```

Comment:
Why "apex-site" folder from the build? The documentation was already created an published (e.g. http://apex.apache.org/docs/apex-3.4/) and now all that remains to do is to create a symlink to make it available under http://apex.apache.org/docs/apex/ ?